$NetBSD: patch-unix_Makefile,v 1.2 2017/11/29 15:13:32 hauke Exp $

cpp(1) only accepts one file for input, we want cc -E here.

Set up for pkgsrc install

--- unix/Makefile.orig	2017-08-23 01:50:11.000000000 +0000
+++ unix/Makefile
@@ -73,6 +73,7 @@ LDFLAGS_ARCH = -Wl,-map,$@.map -Wl,-dead
 else
 # Use gcc syntax for map file
 LDFLAGS_ARCH = -Wl,-Map=$@.map,--cref -Wl,--gc-sections
+CPP = $(CC) -E
 endif
 LDFLAGS = $(LDFLAGS_MOD) $(LDFLAGS_ARCH) -lm $(LDFLAGS_EXTRA)
 
@@ -193,12 +194,10 @@ test: $(PROG) $(TOP)/tests/run-tests
 
 # install micropython in /usr/local/bin
 TARGET = micropython
-PREFIX = $(DESTDIR)/usr/local
-BINDIR = $(PREFIX)/bin
+BINDIR = $(DESTDIR)$(PREFIX)/bin
 
 install: micropython
-	install -d $(BINDIR)
-	install $(TARGET) $(BINDIR)/$(TARGET)
+	${BSD_INSTALL_PROGRAM} $(TARGET) $(BINDIR)/$(TARGET)
 
 # uninstall micropython
 uninstall:
